### DISCUSSION TITLE
Noteblocks/notes display their note pitch

### DISCUSSION THREAD

**Discussion by waffleeiron** [BedrockTweaks/Files#309] [Ideas]
Note particles ejected from noteblocks display the note they are pitched on
Ideally the noteblock itself can display their pitch too

**Comment by SquatchHunter** [comment]
unsure if possible the same way as Java
a lot of people modify the particles to show the pitch
this may change in future but I believe this was deemed "cant do" for the meantime, will need reinvestigating in future (data driven blocks)

**Reply by alexandervertiz9-lang** [reply]
Has your answer changed?

**Reply by SquatchHunter** [reply]
no, Minecraft hasnt given us any new options in regards to changing vanilla blocks
